README.md

```diff
@@ -18,7 +18,13 @@ requires some additional development tools, which are included below.
 ## Common
 
 ```
-pip install CppHeaderParser==2.7.4 meson==1.7.0 PyYAML==6.0.2 toml libtoml tomli
+pip install CppHeaderParser==2.7.4 meson==1.7.0 PyYAML==6.0.2
+```
+
+Python 3.10 further require `tomli`:
```

Comment:
grammar

```suggestion
Python 3.10 also requires `tomli`:
```

This would be easier to capture in a requirements.txt file though, where we can be exact _and_ have just a single pip install command.

Reply:
Yes, a `requirements.txt` is what I would go for next but I wanted to at least fix the README in the meantime.